tutorials/napps/switch_l3_part1.rst

```diff
@@ -111,18 +107,17 @@ Create an L3 switching table
 ============================
 
 First of all, you will create an L3 table for each switch that connects to the
-controller. When a new switch is connected, *Kytos* generates a ``core.switch.new``
+controller. When a new switch is connected, *Kytos* generates a ``of_core.handshake.completed``
```

Comment:
@krishna4041 in the "Creating your NApp" section, the username it's referring to the one registered in the NApp server, could you adapt the phrase to explain that i's supposed to be the same as their GitHub username, like you did for the previous tutorial?

"Use your <username> (the one you have just registered) as the username and 'of_l3ls' as the NApp name, as follows"

Reply:
Made changes.